Yank/experiment.py

```diff
@@ -716,11 +716,16 @@ def run_experiments(self):
 
             # Cycle between experiments every switch_experiment_interval iterations
             # until all of them are done.
-            completed = [False]  # This is just to run the first iteration.
+            completed = [False for _ in range(len(all_experiments))]
```

Comment:
You could have just said `[False] * len(all_experiments)` here, but thats just list compression.